charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -83,6 +83,100 @@ aggregationRule:
       gardener.cloud/role: project-member
 rules: []
 
+# Cluster role granting viewer permissions for the resources in the gardener API group 
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:viewer
+  labels:
+    gardener.cloud/role: viewer
+rules:
+- apiGroups:
+  - core.gardener.cloud
+  - seedmanagement.gardener.cloud
+  - dashboard.gardener.cloud
+  - settings.gardener.cloud
+  - operations.gardener.cloud
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - namespaces
+  - resourcequotas
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - events.k8s.io
+  resources:
+  - events
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterroles
+  - clusterrolebindings
+  - roles
+  - rolebindings
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  - validatingwebhookconfigurations
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apiregistration.k8s.io
+  resources:
+  - apiservices
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - get
+  - list
+  - watch
+
+# Aggregated cluster role specifying permissions for the users with view access for a Gardener landscape.
+# IMPORTANT: You need to define a corresponding ClusterRoleBinding binding specific users
+#            to this ClusterRole on your own (users bound to this role have viewer access to
+#            the Garden system).
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:system:viewers
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      gardener.cloud/role: viewer
+  - matchLabels:
+      gardener.cloud/role: project-viewer
+rules: []
+
+
```

Comment:
You can remove one of the empty lines here.

Reply:
Second empty row removed